src/bin/ion/commands/beta/generate/utils.rs

```diff
@@ -116,26 +201,6 @@ pub enum IonSchemaType {
     SchemaDefined(String), // A user defined schema type
 }
 
-impl IonSchemaType {
-    /// Maps the given ISL type name to a target type
-    pub fn target_type(&self, language: &Language) -> &str {
```

Comment:
(🗺️ PR tour) This has been moved to `target_type()` in `JavaLanguage` and `RustLanguage`.